test/parallel/test-whatwg-transformstream.js

```diff
@@ -186,3 +186,18 @@ class Source {
     inspect(controller, { depth: 0 }),
     /TransformStreamDefaultController \[/);
 }
+
+{
+  Object.defineProperty(Object.prototype, 'type', {
+    get: common.mustNotCall()
+  });
+
+  new TransformStream({
+    transform(chunk, controller) {
+      controller.enqueue(chunk);
+    },
+    flush(controller) {
+      controller.terminate();
+    }
+  });
```

Comment:
```suggestion
  });
  
  delete Object.prototype.type;
```

Reply:
used `Reflect.deleteProperty()` here since direct delete seems to cause an error

Reply:
Both should be equivalent IIRC. Don’t you need to specify `configurable: true` to get rid of the error?

Reply:
Yes using `configurable: true` solves this 
fixed